lib/mobility-core/src/Kernel/External/Notification/FCM/Flow.hs

```diff
@@ -210,25 +216,46 @@ sendMessage ::
   ) =>
   FCMConfig ->
   FCMRequest a b ->
+  m () ->
   Text ->
   m ()
-sendMessage config fcmMsg toWhom = fork desc $ do
+sendMessage config fcmMsg action toWhom = fork desc $ do
   logTagInfo fcm $ "Message to be sent to the person: " <> show (Aeson.encode fcmMsg)
   authToken <- getTokenText config
   case authToken of
     Right token -> do
       let fcmUrl = config.fcmUrl
       res <- callAPI fcmUrl (callFCM (Just $ FCMAuthToken token) fcmMsg) "sendMessage" fcmSendMessageAPI
       case res of
-        Right _ -> logTagInfo fcm $ "message sent successfully to a person with id " <> toWhom
-        Left x -> logTagError fcm $ "error while sending message to person with id " <> toWhom <> " : " <> show x
-    Left err -> do
-      logTagError fcm $ "Auth token fail error while sending message to person with id " <> toWhom <> " : " <> show err
+        Right _ -> logTagInfo fcm $ "Message sent successfully to a person with id " <> toWhom
+        Left clientError -> do
```

Comment:
`logTagError fcm $ "error while sending message to person with id " <> toWhom <> " : " <> show clientError`
Should we keep this log at top so even if any failure happens we get this log atleast ?